### PR TITLE
Update fits.py

### PR DESCRIPTION
**Pull request by balbinot**
Fix bug while plotting and manipulating df coming from FITS files

### CODE REVIEW

File: packages/vaex-astro/vaex/astro/fits.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 class FitsBinTable(DatasetMemoryMapped):
     def __init__(self, filename, write=False, fs_options={}, fs=None):
+	snake_name='fits'
```
Comment (maartenbreddels) [inline packages/vaex-astro/vaex/astro/fits.py:22]:
```suggestion
    snake_name='fits'
    def __init__(self, filename, write=False, fs_options={}, fs=None):
```
The snake name should be at the class level

Comment (maartenbreddels) [reply packages/vaex-astro/vaex/astro/fits.py:22]: I think it also needs the @vaex.dataset.register decorator.

**Comment by balbinot** [comment]
It seems to work well with or without the decorator.

The missing decorator is also in the Gadget class, which I am not sure it is used still. 

https://github.com/vaexio/vaex/commit/b171bcdce68e72589996f2af568a4bb0374b49c9

**Comment by maartenbreddels** [comment]
Ok, this is good enough for now, thanks!

**Pull request merged**